src/test/java/org/opensearch/search/relevance/SearchRelevancePluginIT.java

```diff
@@ -16,7 +17,7 @@
 
 public class SearchRelevancePluginIT extends OpenSearchRestTestCase {
 
-    public void testPluginInstalled() throws IOException {
+    public void testPluginInstalled() throws IOException, ParseException {
```

Comment:
I think we still want the BWC version to be 2.6.0 (or 2.7.0?), for backward compatibility testing.

For the quickstart script, it doesn't make sense to point to a future release, since the various links will be broken until there are artifacts to point to.